.github/workflows/release.yaml

```
@@ -60,12 +78,14 @@ jobs:
           vcpkgDirectory: ${{ github.workspace }}/.vcpkg
 
       - name: Install Qt
-        uses: jurplel/install-qt-action@v4
-        with:
-          arch: ${{ matrix.config.arch }}
-          modules: ${{ matrix.config.qt_modules }}
-          version: ${{ matrix.config.qt_version }}
-          cache: true
+        run: |
```

actionlint (1.7.12)

[error] property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)